lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/LibertyDiagnosticParticipant.java

```diff
@@ -94,39 +98,57 @@ private void validateFeature(DOMDocument domDocument, List<Diagnostic> list, DOM
         }
     }
 
-    private void monitorConfigFiles(DOMDocument domDocument, List<Diagnostic> list, DOMNode node) {
+    /**
+     * Location checks:
+     * 1) Relative path; 2) Server config dir; 
+     * 3) Absolute path; 4) Web server
+     * 
+     * Checks in method: 1), 3)
+     * 2) performed in isConfigXMLFile
+     * 4) not yet implemented/determined
+     */
+    private void validateIncludeLocation(DOMDocument domDocument, List<Diagnostic> list, DOMNode node) {
         String locAttribute = node.getAttribute("location");
         if (locAttribute == null) {
             return;
         }
+        // skip diagnostic for not yet implemented behaviors/checks (URL + vars)
+        if (locAttribute.startsWith("http") || locAttribute.contains("$")) {
+            return;
+        }
 
         DOMNode locNode = node.getAttributeNode("location");
         Range range = XMLPositionUtility.createRange(locNode.getStart(), locNode.getEnd(), domDocument);
-
         if (!locAttribute.endsWith(".xml")) {
-            String message = "The specified file is not an XML file.";
+            String message = "The specified resource is not an XML file.";
             list.add(new Diagnostic(range, message, DiagnosticSeverity.Warning, "liberty-lemminx"));
             return;
         }
 
-        String docURIString = domDocument.getDocumentURI().replace(File.separator, "/");
-        locAttribute = locAttribute.replace(File.separator, "/");
-        File configFile = locAttribute.startsWith("./") ? 
-                new File(URI.create(docURIString.substring(0, docURIString.lastIndexOf("/") + 1))
-                        .resolve(locAttribute).normalize()) :
-                new File(locAttribute);
-
+        String optString = node.getAttribute("optional");
+        boolean required = optString == null ? false : optString.equals("false");
```

Comment:
Do we need to worry about trimming the attribute value that we get back before comparison?

Reply:
I was talking about the `optional` attribute value which is `true` or `false`. Not sure what `xml` has to do with it?

Reply:
We won't need to worry about values for `optional` as the schema defines it

Reply:
Can they specify `FALSE`? You said the schema defines the values, but does that mean we won't get anything besides `true` or `false` here?